src/tempestApi.ts

```diff
@@ -101,4 +118,52 @@ export class TempestApi {
 
   }
 
+  public async getTempestBatteryLevel(): Promise<number> {
+
+    const device_id = await this.getTempestDeviceID();
```

Comment:
I would recommend moving this method to removing having to make 2 API calls and instead make the Device ID an input parameter to this function.

Reply:
`getStationCurrentObservation`, `getTempestBatteryLevel` and `getTempestDeviceId` each make use of different API calls to obtain the required information. 

As a result, the `getTempestDeviceId` method is required. Understand that as the tempest_device_id value is static and should only be called once when plugin is initialized.  Additional updates will be required and will incorporate in v2.1.8 (other changes are coming).